### DISCUSSION TITLE
Make it possible to print the program

### DISCUSSION THREAD

**Discussion by lsolesen** [astashov/liftosaur#186] [Ideas]
I want to use the app for teaching. However, it would be cool, if it was possible to print an overview of the program.

It could just be this page where the actual kg's was just written as percentages. That would be a quick way to show the entire program to the class and explain the overview.:

![image](https://github.com/user-attachments/assets/c25c0030-13da-4c68-b3b8-4920ca3e8eb9)

**Comment by gabemaroz** [comment]
You could save the webpage as a PDF and make edits. Or print it out and write it out. This seems like a really niche function.

**Comment by yrinkel** [comment]
The new sharing implementation covers that I think. If yes this can probably be closed now

**Comment by astashov** [comment]
There's now a way to generate program image, which you can print.
